packages/react-components/react-storybook-addon/src/hooks.ts

```diff
@@ -1,20 +1,30 @@
-import { useGlobals as useStorybookGlobals, Args as StorybookArgs } from '@storybook/api';
+import { useGlobals as useStorybookGlobals, Args as StorybookArgs, Parameters } from '@storybook/api';
 import { StoryContext as StorybookContext } from '@storybook/addons';
 
 import { THEME_ID } from './constants';
 import { ThemeIds } from './theme';
 
 export interface FluentStoryContext extends StorybookContext {
   globals: FluentGlobals;
+  parameters: FluentParameters;
 }
 
 /**
- * Extends the storybook globals object to include fluent specific propoerties
+ * Extends the storybook globals object to include fluent specific properties
  */
 export interface FluentGlobals extends StorybookArgs {
   [THEME_ID]?: ThemeIds;
 }
 
+/**
+ * Extends the storybook parameters object to include fluent specific properties
+ */
+export interface FluentParameters extends Parameters {
+  dir?: 'ltr' | 'rtl';
+  fluentTheme?: ThemeIds;
+  isVrTest?: boolean;
```

Comment:
**suggestion:** boolean flags doesn't scale well. what about 

```diff
-isVrTest?: boolean;
+context?: 'vr-test' | 'default'
```

Reply:
replaced with `mode` flag instead to prevent confusion with storybook `context`: [b5f3788](https://github.com/microsoft/fluentui/pull/25162/commits/b5f378816a1794245bd352c8c4612a222481fac1)